javascript/javascript3/week1/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Api</title>
+  </head>
+  <body>
+    <h1>Books</h1>
+    <p>
+      Click <a href="https://www.abibliadigital.com.br/en">here</a> if you want
+      to go to the site of api
+    </p>
+
+    <p>
+      This api is for books <br />
+
+      which You can choose and the name of the book and The name of the author
+      and more.
+    </p>
+    <p>
+      The api respondes with (array). <br />
+      <strong>The structur looks like this:</strong>
+    </p>
+    <span
+      >[{"abbrev":{"pt":"gn","en":"gn"},"author":"Moisés","chapters":50,"group":"Pentateuco","name":"Gênesis","testament":"VT"}
+      <br />
+      {"abbrev":{"pt":"ex","en":"ex"},"author":"Moisés","chapters":40,"group":"Pentateuco","name":"Êxodo","testament":"VT"}
+      <br />
+      {"abbrev":{"pt":"lv","en":"lv"},"author":"Moisés","chapters":27,"group":"Pentateuco","name":"Levítico","testament":"VT"}</span
+    >
+
+    <p>
+      The api is simply the point of contact between a specific development
+      environment and the developers, so that it allows these developers to take
+      advantage of the services of this environment without having to build
+      everything from scratch, and in general the goal of the api is to hide the
+      details.
```

Comment:
While I think I know what you mean, your description is not entirely accurate. REST APIs don't necessarily relate to the development environment you're working in. Instead, the API is the point of contact between different programs. Like your webbrowser and a webserver. Now, that being said, you can have different APIs for different development environments, like testing and production. 

It is correct that APIs enable you to use a ready-made service without writing it yourself or knowing anything about how it works in the background. For example, for retrieving data from the books API, you don't need to know how the information about books is stored or updated on the APIs side of things.